.rubocop_todo.yml

```diff
@@ -66,6 +66,8 @@ Metrics/MethodLength:
 # Configuration parameters: CountComments.
 Metrics/ModuleLength:
   Max: 267
+  Exclude:
+    - shoes-core/lib/shoes/dsl.rb
```

Comment:
There might or might not be an issue for this, But I'd actually like to slice the dsl file up into multiple modules... like I dunno `Art`, `TextBlock` and other related/grouped methods. We can discuss some other place :)